packages/xstate-wallet/src/store/types.ts

```diff
@@ -130,12 +134,12 @@ export interface Message {
 }
 
 export type ChannelStoredData = {
-  stateVariables: Record<string, StateVariables>;
+  stateVariables: Array<StateVariablesWithHash>;
   channelConstants: Omit<ChannelConstants, 'challengeDuration' | 'channelNonce'> & {
     challengeDuration: BigNumber | string;
     channelNonce: BigNumber | string;
   };
-  signatures: Record<string, Array<string | undefined>>;
+  signatures: Record<string, Array<SignatureEntry>>;
```

Comment:
Does it make sense to refactor this to just store the `signatures` on the entry of the `stateVariables` array? (so that they are guaranteed to be aligned)